dolphinscheduler-task-plugin/dolphinscheduler-task-sql/src/main/java/org/apache/dolphinscheduler/plugin/task/sql/SqlTask.java

```diff
@@ -83,7 +83,7 @@ public class SqlTask extends AbstractTaskExecutor {
     /**
      * create function format
      */
-    private static final String CREATE_FUNCTION_FORMAT = "create temporary function {0} as ''{1}''";
+    private static final String CREATE_OR_REPLACE_FUNCTION_FORMAT = "create or replace temporary function {0} as ''{1}''";
```

Comment:
I believe the temp function could be reused, if we set the value (kyuubi.engine.share.level) to user in Kyuubi. In this case, there is a long running Spark session and all tasks for same user are going to same Spark session. It's more likely to cause sql tasks with UDF to fail, if we only create function.

Reply:
A long running Spark session will keep its own temp functions instead of destroying them right away. You may get failed message(UDF existed) if we run some sql task with UDF. I believe it may be better to change this creating function query.

Reply:
I am ok with this patch, it is compatible with more cases without breaking the before behavior, but could we add some comments before L86 to tell developers why should we include `replace` in this syntax?

Reply:
It is without breaking the before behavior, Am I right? @SbloodyS

Reply:
> It is without breaking the before behavior, Am I right? @SbloodyS

Yes. From your point of view, I think it's ok too.